creativecloud/scripts/scripts.js

```diff
@@ -172,6 +172,14 @@ const CONFIG = {
   ],
 };
 
+function loadLCPPreloads(miloLibs, loadLink) {
+  const marqueeLcp = document.body.querySelector('main > div:nth-child(1) > .marquee');
+  if (!marqueeLcp) return;
+  loadLink(`${miloLibs}/blocks/marquee/marquee.css`, { rel: 'stylesheet' });
+  loadLink(`${miloLibs}/blocks/marquee/marquee.js`, { as: 'script', rel: 'modulepreload' });
+  loadLink(`${miloLibs}/utils/decorate.js`, { as: 'script', rel: 'modulepreload' });
+}
```

Comment:
@auniverseaway 
The changes currently rely on the assumption that the default authoring of the first block on the page is marquee and it will always be marquee
In a case if there is a marquee on page with mep personalization that changes it to hero-marquee..
marquee js and css would get loaded
but then mep will change it
milo will run as usual and hero-marquee js and css will get loaded
So in this scenario extra call was made to a js or css that was not needed

Reply:
@aishwaryamathuria we want to trial this as part of the performance tiger team, we are closely observing the numbers and want to see if this has any positive benefit.

As Chris mentioned, if this assumption (marquee in the first section) is not true, there isn't a lot of harm.